msteams-platform/graph-api/proactive-bots-&-messages/graph-proactive-bots-&-messages.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Use Microsoft Graph to enable proactive bot installation and messaging in Teams
+description: Describes proactive messaging in Teams and how to implement.
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: Overview
+keywords: teams proactive messaging chat installation Graph
+---
+# Enable proactive bot installation and messaging in Teams with Microsoft Graph(Private Preview)
+
+>[!NOTE]
+> Microsoft Graph private preview permissions may not achieve general availability status and currently should not be used in production.
+
+## What is proactive messaging in Teams?
+
+A proactive messages are initiated by bots to start conversations with a users. They serve many purposes including sending welcome messages, conducting surveys or polls, and sending organization-wide notifications.  Proactive messages in Teams can be delivered as either **ad-hoc** or **dialog-based** conversations:
+
+|ad-hoc proactive message|dialog-based proactive message|
+|-----------------------------|------------------------------------|
+|The bot interjects a message without interrupting the conversation flow. | The bot creates a new dialog thread, takes control of a conversation, delivers the proactive message, closes, and returns control to the previous dialog.|
+
+## What is proactive app installation in Teams?
+
+Before your bot can proactively message a user, it needs to be installed either as a personal app, or in a team where the user is a member. At times,  you may need to proactively message users that have _not_ installed or previously interacted with your app. For instance, the need to message vital information to everyone in your organization. For that scenario you can use the Microsoft Graph API to proactively install your bot for your users and cache the necessary values received by your bot at install.
```

Comment:
I would leave out "and cache the necessary values received by your bot at install" as unnecessary to get the point across.